scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -14,6 +14,7 @@ workflow GvsBulkIngestGenomes {
         String sample_id_column_name = "sample_id"
         String? vcf_files_column_name
         String? vcf_index_files_column_name
+        String? sample_set_name ## currently we only allow for one sample set at a time
```

Comment:
are they ever called anything other than "sample_set" ?!?! is it dependent on the entity?